docs-source/content/release-notes.md

```diff
@@ -8,6 +8,7 @@ draft: false
 
 | Date | Version | Introduces backward incompatibilities? | Change |
 | --- | --- | --- | --- |
+| January 22, 2021 | v3.1.2 | no | Resolved an issue where the operator failed to start servers where the pods were configured to have an annotation containing a forward slash. |
```

Comment:
Purely, a slight re-wording suggestion but what you have is fine also: Resolved an issue where the operator failed to start servers in which the pods were configured to have an annotation containing a forward slash.

Reply:
+1